src/components/PollOption.vue

```diff
@@ -87,6 +87,16 @@ export default {
       }
     },
   },
+  methods: {
+    iconColor(iconCssVariable) {
```

Comment:
https://vuejs.org/api/sfc-css-features.html#v-bind-in-css unsterstützt offenbar beliebiges JS und nicht nur Computed Properties. Deswegen habe ich die beiden Funktionen jetzt mal zu einer zusammengefasst.

Reply:
Ah nice! Nur zum Verständnis: Können Computer Properties keine Argumente annehmen?

Reply:
Genau (zumindest in Vue 3, vermutlich aber auch 2). In der Doku steht nichts zu Argumenten, ESLint meckert wenn man computed properties mit Klammern verwendet und aus meiner Berufserfahrung macht man auch ganz klar diese Trennung, dass man mit Argumenten ausschließlich Methoden nutzt.

Ich habe es gerade trotzdem praktisch ausprobiert: Man bekommt die Argumente einfach nicht übergeben, sondern stattdessen die aktuelle Komponenteninstanz (also `this`) als einzigen Parameter.